templates/views/contact.pug

```diff
@@ -0,0 +1,32 @@
+extends ../layouts/base
+
+block content
+  #contact.w-100.absolute.tc.bg-gray.z-1
+    .mxwi6.dib.tc.w-100.pb4
+      h2.tc.light-gray Contact us
+      
+      #address 
+      h3.red Address
+      .silver Gower Str, Bloomsbury, London, WC1E 6BT
+      
+      #phone 
+      h3.red Phone
+      .silver 000 0000 0000
+      
+      h2.tc.light-gray Online
+      
+      #form.dib.w-100.pr2
+        form(method='POST', action=`http://formspree.io/${email}`)
+          input(type='text', name='name', placeholder='You name').dib.w-100.bn.bb.bg-light-silver.br2.ma1.f5.pa2.white
```

Comment:
also, i would prefer to have explicit labels on the form for accessibility and all that. Even if we hide them.